src/dymaptic.GeoBlazor.Core/Components/Widgets/BasemapLayerListWidget.cs

```diff
@@ -0,0 +1,85 @@
+﻿using dymaptic.GeoBlazor.Core.Components.Widgets.LayerList;
+using Microsoft.AspNetCore.Components;
+using Microsoft.JSInterop;
+using System.Text.Json.Serialization;
+
+
+namespace dymaptic.GeoBlazor.Core.Components.Widgets;
+/// <summary>
+///     The ListItem class represents one of the operationalItems in the LayerListViewModel. In the LayerList widget UI, the list item represents a layer displayed in the view. It provides access to the associated layer's properties, allows the developer to configure actions related to the layer, and allows the developer to add content to the item related to the layer.
+///     <a target="_blank" href="https://developers.arcgis.com/javascript/latest/api-reference/esri-widgets-BasemapLayerList.html">ArcGIS JS API</a>
+/// </summary>
+public class BasemapLayerListWidget : LayerListWidget
+{    
+    /// <inheritdoc />
+    [JsonPropertyName("type")]
+    public override string WidgetType => "basemapLayerList";
+
+    /// <summary>
+    ///     A delegate to implement a custom handler for setting up each <see cref="ListItem"/>.
+    ///     Function must take in a ListItem and return a Task with the same (updated) item.
+    /// </summary>
+    [Parameter]
+    [JsonIgnore]
+    public Func<ListItem, Task<ListItem>>? OnBaseListItemCreatedHandler { get; set; }
```

Comment:
Interesting, you created `BaseListItem`, but in here you're always using `ListItem` (which is better).

Reply:
adjusted to listitem

Reply:
These are list items, but there was a display issue when both were encompassed with one set of listitem methods.  The solution to display the different types in separate portions of layerlist table and to view the labels, required different sets of these list item functions.